packages/system-tests/assets/sfdx-simple/.vscode/settings.json

```diff
@@ -1,3 +1,12 @@
 {
-  "extensions.autoUpdate": false
+  "extensions.autoUpdate": false,
```

Comment:
Don't check this in. This should be reverted.

Reply:
Ah ok.  It's weird because the file is already checked in, but vscode modified it...it looks like people have been battling this file for a while.  

I can revert it to it's previous state and add it to .gitignore?
Or I could completely remove it and add it to .gitignore and make it go away forever?

Here's it's git history
![screen shot 2019-01-15 at 3 30 14 pm](https://user-images.githubusercontent.com/599418/51214247-939d2f80-18da-11e9-9ade-4eb2daeba93c.png)

Reply:
You can add it to .gitignore. 
Don't remove it.